### DISCUSSION TITLE
Lesson 7: Command Line Error -> No tests match the provided pattern even with test defined in test file.

### DISCUSSION THREAD

**Discussion by beeski00** [Cyfrin/foundry-full-course-cu#881] [Q&A] [answered]
Hey All,

I am on lesson 7 testing the PriceFeed and I am recieving an error saying "No tests match the provided pattern. However, I have defined my test in my test file FundMeTest.t.sol

```
`function testPriceFeedVersionIsAccurate() public {
        uint256 version = fundMe.getVersion();
        assertEq(version, 4);
    }`
    
   ```

   Command Line:
   ```
MacBook-Air:foundry-fund-me-f23 $ forge test --mt testPriceFeedVersionIsAccurate -vv --fork-url $Sepolia_RPC_URL
[⠒] Compiling...
No files changed, compilation skipped

No tests match the provided pattern:
        match-test: `testPriceFeedVersionIsAccurate`
```
 I copied and pasted the test name ... any thoughts or insights is much appreciated. Thank you in advance.

**Comment by 0xusmanf** [comment, marked as answer]
Make sure your FundMeTest.t.sol is in the test folder and you have saved the file after writing the test.

If you still get the error then share the complete FundMeTest.t.sol file.
